beginner_source/deeplabv3_on_android.rst

```diff
@@ -1,234 +1,10 @@
 Image Segmentation DeepLabV3 on Android
 =================================================
 
-**Author**: `Jeff Tang <https://github.com/jeffxtang>`_
+PyTorch Mobile is no longer actively supported. Please check out ExecuTorch.
```

Comment:
The spellcheck still doesn't like it, let's make sure we spell ExecuTorch everywhere and add it to the https://github.com/pytorch/tutorials/blob/main/en-wordlist.txt